UT4MasterServer/Controllers/UnrealTournamentGameController.cs

```diff
@@ -120,7 +123,9 @@ public async Task<IActionResult> SetAvatarAndFlag(string id, [FromQuery] string
         if (user.Session.AccountID != EpicID.FromString(id))
             return Unauthorized();
 
-        JObject obj = JObject.Parse(await Request.BodyReader.ReadAsStringAsync(1024));
+		// TODO: Permission: "Sorry your login does not posses the permissions 'ut:profile:{id_from_params}:commands ALL' needed to perform the requested operation"
+
+		JObject obj = JObject.Parse(await Request.BodyReader.ReadAsStringAsync(1024));
```

Comment:
indentation looks weird here on github. but im guessing its just because the rest of the code has spaces and not tabs.